bring-it-up/bring-it-up
# m1-service-card

## Description
#### Summary
- Create service card to display service information

#### Changes
- Make firstDropdown.js file in components folder
- Add styling to index.css
- Change small error in firstDropdown (add unique keys to each Dropdown item)
#### How to test this PR
1. Go to App.tsx
2. Use ServiceCard component, passing in appropriate props (for now as it is not connected to BE)
3. npm start
4. Service card appears at the top of the screen

#### Notes
Some spacing still needs to be fixed

## Screenshots:

![image](https://user-images.githubusercontent.com/73853606/175847976-a3aaf3e0-749b-407a-b9de-bb6267d125d3.png)
